lib/login/view/login_page.dart

```diff
@@ -11,16 +12,21 @@ class LoginPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: const Text('Login')),
-      body: Padding(
-        padding: const EdgeInsets.all(8.0),
+      backgroundColor: GreenHouseColors.green,      
+      body: Container(
+        decoration: BoxDecoration(
+          image: DecorationImage(
+            image: AssetImage('images/greenhouse.jpg'),
+            fit: BoxFit.cover,
+          ),
+        ),
         child: BlocProvider(
           create: (_) => LoginCubit(
             context.repository<AuthenticationRepository>(),
           ),
           child: LoginForm(),
         ),
-      ),
+      ),            
     );
   }
 }
```

Comment:
Always add a new line to the end of a file see [Stackoverflow](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).